samples/custom_dispatch/cpu/mlp_plugin/mlp_torch.mlir

```diff
@@ -0,0 +1,81 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-preprocessing-apply-pdl-patterns{patterns-file=%p/mlp_torch_spec.pdl.mlir})" %s | \
```

Comment:
Could we add another flag like `--input-preprocessing-pipeline` that uses the same preprocessing custom pipeline mechanism, but happens before input conversion and doesn't need to use iree-opt like this?

Reply:
Yeah... maybe need that, but punting till we actually need this.